python_modules/airline-demo/airline_demo/solids.py

```diff
@@ -162,29 +161,6 @@ def thunk(info):
     return info.config
 
 
-# Maybe parametrize this to make_thunk(type)
-@solid(
-    name='thunk_list',
-    config_field=Field(types.List(types.Any), description='The list value to output.'),
-    description='No-op solid that simply outputs its single list config value.',
-    outputs=[OutputDefinition(types.List(types.Any), description='The list passed in as config.')]
-)
-def thunk_list(info):
-    '''Output the config vakue.
-
-    Especially useful when constructing DAGs with root nodes that take inputs which might in
-    other dags come from upstream solids.
-
-    Args:
-        info (ExpectationExecutionInfo)
-
-    Returns:
-        list:
-            The config value passed to the solid.
-    '''
-    return info.config
-
-
 @solid(
```

Comment:
following this line of though, presumably we eventually have an input type here which takes config

Reply:
thunks should be totally unnecessary if the type is an input which is Configurable